py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -800,9 +808,10 @@ def eth_sign_msg(
             if self.debug:
                 print(f"Antiklepto nonce verification PASSED")
 
-            return signature
+            return modify_recid(signature)
 
-        return self._eth_msg_query(request, expected_response="sign").sign.signature
+        signature = self._eth_msg_query(request, expected_response="sign").sign.signature
+        return modify_recid(signature)
```

Comment:
when reading here without knowing the body of `modify_recid`, it's unclear what it does. consider renaming to something like `format_as_uncompressed`.

Reply:
done